packages/commonwealth/server/routes/topics/create_topic_handler.ts

```diff
@@ -13,7 +19,28 @@ export const createTopicHandler = async (
 ) => {
   const { user, chain, body } = req;
 
-  const topic = await controllers.topics.createTopic({ user, chain, body });
+  const validationSchema = z.object({
+    name: z.string().optional(),
+    description: z.string().optional(),
+    token_threshold: z.string().optional(),
+    featured_in_sidebar: z.coerce.boolean().optional(),
+    featured_in_new_post: z.coerce.boolean().optional(),
+    default_offchain_template: z.string().optional(),
+    chain_id: z.string().optional(),
+  });
+
+  const validationResult = validationSchema.safeParse(body);
+  if (validationResult.success === false) {
+    throw new AppError(
+      `${Errors.ValidationError}: ${validationResult.error.message}`
```

Comment:
How does this work if multiple schema fields fail? For example in the docs they have this example for error checking in general:
https://github.com/colinhacks/zod/blob/master/ERROR_HANDLING.md#a-demonstrative-example

and this example (using safe-parse) for forms:
https://github.com/colinhacks/zod/blob/master/ERROR_HANDLING.md#error-handling-for-forms